### ISSUE TITLE
Export maskValue method

### ISSUE THREAD

**Issue by sawyerh** [CMSgov/design-system#288]
## Description

As a Developer, I would like to submit form data to my server as an unmasked value to make validation easier, but when I re-display the value in a read-only state (i.e. in `<p>` tag), I would like to display the masked version.

Currently, I can do the first part, unmasking the value using the design system's exported `unmaskValue` method. However, there isn't an exported method to mask the value, though [it exists](https://github.com/CMSgov/design-system/blob/master/packages/core/src/components/TextField/Mask.jsx#L110-L129).

TLDR: Export `maskValue` along with the already [exported `unmaskValue`](https://github.com/CMSgov/design-system/blob/master/packages/core/src/components/TextField/TextField.jsx#L8)

**Comment by benkutil** [comment]
@sawyerh will get this into the backlog. In the meantime, if you’d like to propose a PR, would appreciate that as well.

**Comment by line47** [comment]
@pwolfert @hannaliebl or @1Copenut do you all know the state of this? Should we still consider this work?

**Comment by pwolfert** [comment]
At some point since this issue was reported, we started exporting that function 👍
